InstructorFeedbackEditPageUiTest failing on dev server

Fails at different places at different runs. Possibly due to delay in loading visibility preview message.

```
java.lang.AssertionError: Expected the receiving student to be able to see response, but was 
	at org.testng.AssertJUnit.fail(AssertJUnit.java:59)
	at org.testng.AssertJUnit.assertTrue(AssertJUnit.java:24)
	at teammates.test.cases.BaseTestCase.assertTrue(BaseTestCase.java:160)
	at teammates.test.cases.ui.browsertests.InstructorFeedbackEditPageUiTest.testEditQuestionAction(InstructorFeedbackEditPageUiTest.java:330)
	at teammates.test.cases.ui.browsertests.InstructorFeedbackEditPageUiTest.testGeneralQuestionOperations(InstructorFeedbackEditPageUiTest.java:101)
	at teammates.test.cases.ui.browsertests.InstructorFeedbackEditPageUiTest.allTests(InstructorFeedbackEditPageUiTest.java:83)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:606)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1224)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1149)
	at org.testng.TestNG.run(TestNG.java:1057)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:207)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:178)
```

--- a/src/test/java/teammates/test/cases/ui/browsertests/InstructorFeedbackEditPageUiTest.java
+++ b/src/test/java/teammates/test/cases/ui/browsertests/InstructorFeedbackEditPageUiTest.java
@@ -746,6 +746,8 @@ public class InstructorFeedbackEditPageUiTest extends BaseUiTestCase {
         feedbackEditPage.selectRecipientToBe(FeedbackParticipantType.STUDENTS, 1);
         feedbackEditPage.enableVisibilityOptions(1);
         feedbackEditPage.clickGiverNameVisibilityCheckBox("STUDENTS", 1);
+        WebElement visibilityMessage1 = browser.driver.findElement(By.id("visibilityMessage-1"));
+        feedbackEditPage.waitForElementVisibility(visibilityMessage1);
         assertTrue("Visibility message does not correspond to visibility options",
                    feedbackEditPage.getVisibilityMessage(1).contains("Other students in the course can see your response, "
                                                                      + "and your name, but not the name of the recipient"));
@@ -907,20 +909,20 @@ public class InstructorFeedbackEditPageUiTest extends BaseUiTestCase {
         feedbackEditPage.selectRecipientsToBeInstructors();
         feedbackEditPage.clickAddQuestionButton();
 
+        ______TS("preview as instructor");
+
+        feedbackEditPage.waitForElementPresence(By.id("button_preview_instructor"));
+        FeedbackSubmitPage previewPage = feedbackEditPage.clickPreviewAsInstructorButton();
+        previewPage.verifyHtmlMainContent("/instructorFeedbackSubmitPagePreview.html");
+        previewPage.closeCurrentWindowAndSwitchToParentWindow();
+
         ______TS("preview as student");
 
-        FeedbackSubmitPage previewPage;
+        feedbackEditPage.waitForElementPresence(By.id("button_preview_student"));
         previewPage = feedbackEditPage.clickPreviewAsStudentButton();
         previewPage.verifyHtmlMainContent("/studentFeedbackSubmitPagePreview.html");
         previewPage.closeCurrentWindowAndSwitchToParentWindow();
 
-        
-        ______TS("preview as instructor");
-
-        previewPage.waitForElementPresence(By.id("button_preview_instructor"));
-        previewPage = feedbackEditPage.clickPreviewAsInstructorButton();
-        previewPage.verifyHtmlMainContent("/instructorFeedbackSubmitPagePreview.html");
-        previewPage.closeCurrentWindowAndSwitchToParentWindow();
     }
 
     private void testDoneEditingLink() {

